google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/InsertAllRequest.java

```diff
@@ -50,9 +50,11 @@
 
   /**
    * A Google Big Query row to be inserted into a table. Each {@code RowToInsert} has an associated
-   * id used by BigQuery to detect duplicate insertion requests on a best-effort basis. Please
-   * notice that data for fields of type {@link LegacySQLTypeName#BYTES} must be provided as a
-   * base64 encoded string.
+   * id used by BigQuery to detect duplicate insertion requests on a best-effort basis.
+   *
+   * <p>To ensure proper serialization of numeric data, it is recommended to supply values using a
```

Comment:
per google tech writing style, avoid passive voice; that is, "it is recommended to supply values" --> "supply values"